server/bin/pbench-remote-sync-package-tarballs

```diff
@@ -47,4 +49,4 @@ if [ -z "${remote_opt}" ]; then
     exit 1
 fi
 
-ssh ${remote_host} "${remote_opt}/bin/pbench-sync-package-tarballs" > ${output_tar_ball}
+ssh ${remote_host} "${remote_opt}/bin/pbench-sync-package-tarballs" | tee ${ssh_error} ${output_tar_ball} >/dev/null
```

Comment:
This is going to break the current method of use for `pbench-remote-sync-package-tarballs`.

The way this works today is by creating a tar ball on the remote side via `tar -cf -`, where the tar ball contents are sent to `stdout` which this script captures to a file given by the caller of `pbench-remote-sync-package-tarballs`.

I think all we have to do is capture `stderr` to a file separately, like:

```
ssh ${remote_host} "${remote_opt}/bin/pbench-sync-package-tarballs" > ${output_tar_ball} 2> ${errors}
```

The `${errors}` file could be specified as a third argument to `pbench-remote-sync-package-tarballs`.